modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -368,67 +444,217 @@ private void _testSystemObjectRelationshipManyToMany() throws Exception {
 				objectRelationshipTypeException.getMessage());
 		}
 
+		try {
+			_objectRelationshipLocalService.addObjectRelationship(
+				TestPropsValues.getUserId(),
+				_systemObjectDefinition1.getObjectDefinitionId(),
+				_objectDefinition1.getObjectDefinitionId(),
+				RandomTestUtil.randomLong(),
+				ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				StringUtil.randomId(),
+				ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
+		}
+		catch (ObjectRelationshipParameterObjectFieldIdException
+					objectRelationshipParameterObjectFieldIdException) {
+
+			Assert.assertEquals(
+				"Object relationship type " +
+					ObjectRelationshipConstants.TYPE_MANY_TO_MANY +
+						" does not allow a parameter object field ID",
+				objectRelationshipParameterObjectFieldIdException.getMessage());
+		}
```

Comment:
I think Brian usually sorts by exception name, so this would have to come before the `ObjectRelationshipTypeException`

Reply:
Didn't know that. I'll make this change.

Reply:
Done

Reply:
Done again